### PR TITLE
Remove deprecated lifecycle method

### PR DESCRIPTION
**Pull request by courtyenn**

### CODE REVIEW

File: src/multi-selector.css [hunk 1]
```diff
@@ -32,6 +32,11 @@
   overflow-y: auto;
 }
 
+/* Do not remove. This class is targeted with JS */
```
Comment (blittle) [inline src/multi-selector.css:35]: Why would removing the css rule effect targeting with JS?

File: src/multi-selector.js [hunk 2]
```diff
@@ -103,16 +103,23 @@ export default class MultiSelector extends React.Component {
       lastModifiedItem: null,
       invalid: false,
     };
+
+    document.addEventListener('click', this.close);
```
Comment (blittle) [inline src/multi-selector.js]: NIT - Why was this moved into the constructor? Semantically  it might make more sense inside `componentDidMount`

Comment (courtyenn) [reply src/multi-selector.js]: I can move it to `componentDidMount`

File: src/multi-selector.css (hunk 1 above)
Comment (courtyenn) [reply src/multi-selector.css:35]: This specific CSS class is being targeted

Comment (courtyenn) [reply src/multi-selector.css:35]: https://github.com/CanopyTax/cpr-multiselect/blob/a5d135246a6784f967f1e23091713805cf57110f/src/multi-selector.js#L414

This is the line that needs this CSS rule

**Comment by blittle** [comment]
🐫
